src/components/scenes/TransactionsExportScene.tsx

```diff
@@ -142,7 +148,8 @@ class TransactionsExportSceneComponent extends React.PureComponent<Props, State>
   handleAndroidToggle = () => {
     this.setState(state => ({
       isExportCsv: !state.isExportCsv,
-      isExportQbo: !state.isExportQbo
+      isExportQbo: !state.isExportQbo,
+      isExportBitwave: !state.isExportBitwave
```

Comment:
The Android toggles are messed up. We need to do the following:

```ts
  handleQboToggle = () => {
    if (android) {
      this.setState({ isExportQbo: true, isExportCsv: false, isExportBitwave: false })
   } else { 
     this.setState(state => ({ isExportQbo: !state.isExportQbo }))
   }
  }
```

And likewise for the other 2 handlers. Then in the `renderAndroidSwitches`, we use these 3 handlers, instead of the generic `handleAndroidToggle`.

This is because Android can only choose one option at a time. With two options, it doesn't really matter which is which, but with 3 options, it suddenly matters which switch you choose, so we need separate handlers.

Reply:
good catch. will do.